src/env/mpicc.bash.in

```diff
@@ -102,6 +102,7 @@ interlib_deps=yes
 static_mpi=no
 showinfo=""
 delay_libs=
+mpi_abi=no
```

Comment:
How about make `mpicc` accept an option `-mpi-abi`? Then we can simply install `mpicc-abi` as an alias to `mpicc`. We can turn the `mpi-abi` option on by checking the script name, e.g. -
```
if test "${0%_abi}" != "$0" ; then
    mpi_abi=yes
fi
```